src/inverted_index.c

```diff
@@ -980,6 +996,18 @@ int IR_Read(void *ctx, RSIndexResult **e) {
       continue;
     }
 
+    if (ir->skipMulti) {
```

Comment:
Do we ever want to not `skipMulti`?

Reply:
Yes, when we need all the values, such as when splitting entries among ranges in `NumericRange_Split`